libs/base/ui/action/link/src/link.model.ts

```diff
@@ -23,3 +23,8 @@ export const enum ColorType {
   Primary = 'primary',
   Neutral = 'neutral',
 }
+
+export const enum LinkAppearance {
```

Comment:
Why do you need this enum? And why is it related to the ui link?

The initial idea was to add a new option for the content link that will make it look like menu item. It has already had a `button` option, that makes it look like just a button, that is completely strange. I would suggest to add a new option -> `appearance` that will have with possible values:

```
export const enum ContentLinkAppearance {
  Link = 'link',
  Button = 'button',
  MenuLink = 'menuLink',
}
```

ContentLinkAppearance.Link will be the default value for the option, and in the future we will deprecate the `button` option

@dunqan What do you think?

Reply:
I tried putting it in content library originally but then it couldn't be used in the auth `login-link` component because content library is in `domain`, and I get warnings that I'm not allowed to use it unless it's in `base` or `platform`.

Otherwise I can just add the extra button option to fill in for the option we plan to deprecate.

Reply:
We will use `oryx-option` in `login-link` to avoid this problem and keep `ContentLinkAppearance` in the content package.